### DISCUSSION TITLE
Wrong Sensor detected: Soil Moisture Sensor TS0601 instead of QT-07S

### DISCUSSION THREAD

**Discussion by petit5** [Koenkk/zigbee2mqtt#18785] [Devices]
Hi

First warning: I'm totally new to all these topics and I'm not sure if I'm doing it all right here on the forum.... ;-)

Problem:
- I have a "[QT-07S](https://www.zigbee2mqtt.io/devices/QT-07S.html)" type soil moisture sensor.
- I would like to integrate the sensor into my Openhab configuration using Zigbee2MQTT.
- But unfortunately when I paire the sensor of type "[QT-07S](https://www.zigbee2mqtt.io/devices/QT-07S.html)" it is detected as type "[TS0601_soil](https://www.zigbee2mqtt.io/devices/TS0601_soil.html)".
- I **suspect** (not sure) that this is why the humidity value does not work reliably. It always reports either 0% or 100%, but no values in between.
- It seems to be a similar problem to Issue [#5941](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5941), but not the exact same problem...?

Idea:
- I would like to change the type of the sensor in Zigbee2MQTT, or make sure it is detected and paired correctly.
- After that I want to check if the value for humidity is correct or if it is another problem.

Configuration:
- Zigbee2MQTT [version 1.32.2](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.32.2) commit: [1ec1e572](https://github.com/Koenkk/zigbee2mqtt/commit/1ec1e572)

Questions for the experts (you):
- Can I change the type myself in the configuration of Zigbee2MQTT, or is the device type sent by the device itself when pairing and cannot be changed?
- If I can change it myself: Are there any instructions or tips on how I need to do this? As I said, I'm a beginner and would need instructions for dummies ;-)
- Or is this a "bug" in Zigbee2MQTT  [version 1.32.2](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.32.2) that the device is detected incorrectly and needs to be fixed in a follow up version?

Thanks in advance for your answers, looking forward to your feedbacks.

Best regards
Petit5

**Comment by petit5** [comment]
Hi

Does anyone have an idea or a tip for me?

Best regards
Petit5

**Reply by MilesTEG1** [reply]
Hello ,
It seems I have the same problem...
Is there any solution?

**Reply by mabiste** [reply]
I've also have the same problem. I'm running zigbee2mqtt 2.1.0 (via Home Assistant add-on 2.1.0-1).

The QT-07S is found by my SLZB-06, but as 'unsupported':
![grafik](https://github.com/user-attachments/assets/15939c77-772f-4bbb-a78c-1ce1d3a6d3f7)
![grafik](https://github.com/user-attachments/assets/74b07268-8d02-4abd-a062-351b07313905)

On the 'dev console' tab, the generated_external_definition:
```
const definition = {
    zigbeeModel: ['TS0601'],
    model: 'TS0601',
    vendor: '_TZE284_myd45weu',
    description: 'Automatically generated definition',
    extend: [],
    meta: {},
};

module.exports = definition;
```

Anyone who can help to integrate the device?
Thanks!
Marco

**Reply by mabiste** [reply]
Should be solved, see https://github.com/Koenkk/zigbee2mqtt/discussions/26073#discussioncomment-12034052
I will try it out tonight :-)
